server/src/main/scala/actors/DrtSystem.scala

```diff
@@ -281,26 +278,26 @@ case class DrtSystem(actorSystem: ActorSystem, config: Configuration, airportCon
         else LHRFlightFeed()
       case "EDI" => createLiveChromaFlightFeed(ChromaLive).chromaEdiFlights()
       case "LGW" => LGWFeed()
-      case "BHX" => BHXLiveFeed(config.getString("feeds.bhx.soap.endPointUrl").get)
+      case "BHX" => BHXLiveFeed(config.getString("feeds.bhx.soap.endPointUrl").getOrElse(""))
```

Comment:
I think this should probably be getOrElse(throw Exception("Missing BHX Soap Endpoint URL"))

Reply:
yep. done